CoordImporter/Windows/MainWindow.cs

```diff
@@ -27,28 +28,22 @@ public class MainWindow : Window, IDisposable
     public override void Draw()
     {
         ImGui.Spacing();
-        if (ImGui.Button("Import", new Vector2(80, 24)))
+        if (ImGui.Button("Import"))
         {
             plugin.EchoString(textBuffer);
         }
+        
         ImGui.SameLine();
-        ImGui.Dummy(new Vector2(24.0f, 0.0f));
```

Comment:
oh the minimum size is visible in the discord screenshot. the buttons are extremely visible even at the minimum size, so i don't think the problem you describe is possible with this change. however i don't mind there being a space! i only took it out because it seemed unnecessary with the scaling that's happening now

Reply:
If the import button is the only visible one at the minimum size then that's great!